src/appMain/CMakeLists.txt

```diff
@@ -143,6 +143,12 @@ endif()
 add_executable(${PROJECT} ${SOURCES})
 target_link_libraries(${PROJECT} ${LIBRARIES})
 
+add_dependencies(${PROJECT} Policy)
+
+if(ENABLE_LOG)
+  add_dependencies(${PROJECT} install-3rd_party_logger)
```

Comment:
@istoilovgithub the idea is Utils to hide details about loggers etc. and to have once place to edit if in the future we do not depend from this logger etc.